packages/lit-dev-content/site/docs/ssr/client-api.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Lit SSR client API
+eleventyNavigation:
+  key: Client API
+  parent: Server rendering
+  order: 3
+---
+
+{% labs-disclaimer %}
+
+Lit SSR generates static HTML for the browser to parse and paint without any JavaScript. (Browsers that do not have support for Declarative Shadow DOM will require some JavaScript polyfill for Lit components authored to utilize the Shadow DOM.) For pages with static content, this is all that's needed. However, if the page content needs to be dynamic and respond to user interactions, it will need JavaScript to re-apply that reactivity.
+
+How to re-apply that reactivity client-side depends on whether you are rendering standalone Lit templates or utilizing Lit components.
+
+## Standalone Lit Templates
+"Hydration" for Lit templates is the process of having Lit re-associate the expressions of a Lit templates with the nodes they should update in the DOM as well as adding event listeners. In order to hydrate Lit templates, the `hydrate()` method from the `experimental-hydrate` module is provided in the `lit` package. Before you update a server-rendered container using `render()`, you must first call `hydrate()` on that container using the same template and data that was used to render on the server:
+
+```js
+import {render} from 'lit';
+import {hydrate} from 'lit/experimental-hydrate.js';
+import {myTemplate} from './my-template.js';
+// Initial hydration required before render:
+// (must be same data used to render on the server)
+const initialData = getInitialAppData();
+hydrate(myTemplate(initialData), document.body);
+
+// After hydration, render will efficiently update the server-rendered DOM:
+const update = (data) => render(myTemplate(data), document.body);
+```
+
+## Lit components
+To re-apply reactivity to Lit components, the custom element definition needs to be loaded for them to be upgraded, enabling their lifecycle callbacks, and the templates in the components' shadow roots needs to be hydrated.
+
+Upgrading can be achieved simply by loading the component module that registers the custom element. This can be done by loading a bundle of all the component definitions for a page, or may be done based on more sophisticated heuristics where only a subset of definitions are loaded as needed. To ensure templates in `LitElement` shadow roots are hydrated, load the `lit/experimental-hydrate-support.js` module which installs support for `LitElement` to automatically hydrate itself when it detects it was server-rendered with declarative shadow DOM. This module must be loaded before the `lit` module, and therefore any component modules that import `lit`, is loaded to ensure hydration support is properly installed.
```

Comment:
Found this sentence a little hard to parse, might help:
```suggestion
Upgrading can be achieved simply by loading the component module that registers the custom element. This can be done by loading a bundle of all the component definitions for a page, or may be done based on more sophisticated heuristics where only a subset of definitions are loaded as needed. To ensure templates in `LitElement` shadow roots are hydrated, load the `lit/experimental-hydrate-support.js` module which installs support for `LitElement` to automatically hydrate itself when it detects it was server-rendered with declarative shadow DOM. This module must be loaded before the `lit` module is loaded (including any component modules that import `lit`) to ensure hydration support is properly installed.
```